bin/deploy-contracts

```diff
@@ -0,0 +1,82 @@
+#! /bin/bash
+set -e
+
+# select a Geth or Ganache client
+if [ -z "$1" ]; then
+  echo "Please, inform the .env file to be used for deploying!"
+  exit 1
+fi
+
+DEPLOYMENT_ENV_FILE=$1
+
+if [ -z "${DEPLOYMENT_SERVICES}" ]; then
+  DEPLOYMENT_SERVICES=$(cat ${DEPLOYMENT_ENV_FILE} | sed  -n -e 's/^export DEPLOYMENT_SERVICES=//p')
+fi
+
+if [ -z "${DEPLOYMENT_SERVICES}" ]; then
+  DEPLOYMENT_SERVICES="deployer,worker"
+fi
```

Comment:
Unfortunately no, because it first verify if the user passed the env variable via command line, if not the program tries to take it from the conf file, then if the user didn't enter any value for the env var inside the conf file, it will fallback to "deployer,worker". Anyway, I did a change that makes more sense

```
if [ -z "${DEPLOYMENT_SERVICES}" ]; then
  DEPLOYMENT_SERVICES=$(cat ${DEPLOYMENT_ENV_FILE} | sed  -n -e 's/^export DEPLOYMENT_SERVICES=//p')

  if [ -z "${DEPLOYMENT_SERVICES}" ]; then
    DEPLOYMENT_SERVICES="deployer,worker"
  fi
fi
```

Thank you! ;)